### ISSUE TITLE
`updateExpansionAria` called on wrong `this`

### ISSUE THREAD

**Issue by henrikthuesen** [primefaces/primefaces#12128]
### Describe the bug

When using a data table with `rowExpandMode="single"` the `collapseAllRows` function in `datatable.js` is called and on line [3482](https://github.com/primefaces/primefaces/blob/65accbebfa225007353b02a67ca6a182f530a6f2/primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.js#L3482) `updateExpansionAria` is called on `this`, but this is not the correct object. The correct `this` is captured on line 3472, so it should be `$this` instead. 

The current implementation produces the following exception:

```
Uncaught TypeError: this.updateExpansionAria is not a function
    collapseAllRows components.js.xhtml:5641
    jQuery 2
    collapseAllRows components.js.xhtml:5634
    toggleExpansion components.js.xhtml:5518
    bindExpansionEvents components.js.xhtml:4190
    jQuery 8
    bindExpansionEvents components.js.xhtml:4184
    init components.js.xhtml:3446
    Class.extend/f[g]< core.js.xhtml:114
    d core.js.xhtml:114
    createWidget core.js.xhtml:26
    cw core.js.xhtml:26
    <anonymous> line 2 > injectedScript:1
    jQuery 6
    handle components.js.xhtml:509
    doUpdate core.js.xhtml:79
    handle core.js.xhtml:76
    onsuccess components.js.xhtml:502
    b core.js.xhtml:69
    jQuery 6
    send core.js.xhtml:68
    offer core.js.xhtml:57
    handle core.js.xhtml:58
    loadDynamicTab components.js.xhtml:521
    select components.js.xhtml:421
    bindEvents components.js.xhtml:368
    jQuery 8
    bindEvents components.js.xhtml:361
    init components.js.xhtml:321
    Class.extend/f[g]< core.js.xhtml:114
    d core.js.xhtml:114
    createWidget core.js.xhtml:26
    cw core.js.xhtml:26
    <anonymous> line 2 > injectedScript:1
    jQuery 3
    updateElement core.js.xhtml:56
    doUpdate core.js.xhtml:79
    handle core.js.xhtml:76
    b core.js.xhtml:70
    jQuery 6
    send core.js.xhtml:68
    offer core.js.xhtml:57
    handle core.js.xhtml:58
    ab core.js.xhtml:50
components.js.xhtml:5641:15
```

### PrimeFaces edition

None

### PrimeFaces version

14.0.0

### Theme

sage

### JSF implementation

Mojarra

### JSF version

4.0

### Java version

17.0.10

### Browser(s)

Firefox 127.0 macos

**Comment by melloware** [comment]
Thanks for the report and the PR!
